webdriver/tests/bidi/script/call_function/user_activation.py

```diff
@@ -7,37 +7,35 @@
 @pytest.mark.parametrize("user_activation", [True, False])
 async def test_userActivation(bidi_session, top_context, user_activation):
     # Consume any previously set activation.
-    await bidi_session.script.evaluate(
-        expression="""window.open();""",
-        target=ContextTarget(top_context["context"]),
-        await_promise=False)
+    await bidi_session.script.evaluate(expression="""window.open();""",
+                                       target=ContextTarget(
+                                           top_context["context"]),
+                                       await_promise=False)
 
     result = await bidi_session.script.call_function(
-        function_declaration="() => navigator.userActivation.isActive && navigator.userActivation.hasBeenActive",
+        function_declaration=
+        "() => navigator.userActivation.isActive && navigator.userActivation.hasBeenActive",
         target=ContextTarget(top_context["context"]),
         await_promise=True,
         user_activation=user_activation)
 
-    assert result == {
-        "type": "boolean",
-        "value": user_activation}
+    assert result == {"type": "boolean", "value": user_activation}
 
 
 @pytest.mark.asyncio
 @pytest.mark.parametrize("user_activation", [True, False])
 async def test_userActivation_copy(bidi_session, top_context, user_activation):
     # Consume any previously set activation.
-    await bidi_session.script.evaluate(
-        expression="""window.open();""",
-        target=ContextTarget(top_context["context"]),
-        await_promise=False)
+    await bidi_session.script.evaluate(expression="""window.open();""",
+                                       target=ContextTarget(
+                                           top_context["context"]),
+                                       await_promise=False)
 
     result = await bidi_session.script.call_function(
-        function_declaration="() => document.body.appendChild(document.createTextNode('test')) && document.execCommand('selectAll') && document.execCommand('copy')",
+        function_declaration=
+        "() => document.body.appendChild(document.createTextNode('test')); document.execCommand('selectAll'); document.execCommand('copy')",
```

Comment:
This change most likely caused the tests to fail. Why did you replace `&&` with `;`?

Reply:
To make it fit into the max number of lines...(the cutoff was ~4 characters or so)

Reply:
Fixed

Reply:
But the `;` breaks the test. See the results at https://wpt.fyi/results/webdriver/tests/bidi/script/call_function/user_activation.py?diff&filter=ADC&run_id=6265633395703808&run_id=6331982050492416.

Reply:
It's reverted now, I just split the string with '+' as an alternative

Reply:
Can you please make sure the test doesn't fail locally? It's still broken due to: `FAIL message: webdriver.bidi.modules.script.ScriptEvaluateResultException: SyntaxError: missing ) in parenthetical`.

Reply:
Where do you see this? I cannot see it in the logs of the stability jobs.

Reply:
These are the wpt.fyi results for each browser. Just scroll down the list of jobs. Also you should always check those jobs even they appear green. Test failures aren't making those jobs fail.

Reply:
> Also you should always check those jobs even they appear green. Test failures aren't making those jobs fail.

This is surprising. I usually only check the failing jobs. In this case, the stability jobs:

- https://github.com/web-platform-tests/wpt/pull/43096/checks?check_run_id=18682046616
- https://github.com/web-platform-tests/wpt/pull/43096/checks?check_run_id=18682046897

And none of the log there has failures pertaining to this PR (that I see).

Is this the only broken test, or are there more? I'd appreciate if you could tell me how to get the full list.

I understand how to get to the wpt.fyi job: https://github.com/web-platform-tests/wpt/pull/43096/checks?check_run_id=18684017796

From here, where do you go next?

Reply:
It's fixed now!

The mistake was: I split the string with `+` but forgot to change `; -> &&`, my bad.

Reply:
> Is this the only broken test, or are there more? I'd appreciate if you could tell me how to get the full list.
> 
> I understand how to get to the wpt.fyi job: https://github.com/web-platform-tests/wpt/pull/43096/checks?check_run_id=18684017796

As you can see on that page and switching between Chrome and Firefox the common failure was for `user_activation.py`. But it's also `perform_actions/wheel.py` which seems to maybe sporadically fail in Chrome. You can use the [Visual comparison of the results](https://wpt.fyi/results/webdriver/tests/bidi/script/call_function/user_activation.py?diff=&filter=ADC&run_id=5099264931266560&run_id=5187860208025600)] link right under the tables to actually get to the wpt.fyi results where you then can [extend the details for failures and see everything](https://wpt.fyi/results/webdriver/tests/bidi/input/perform_actions/wheel.py?diff&filter=ADC&run_id=5132912409706496&run_id=6289216054493184). And yes, there is a bug as well for `wheel.py`.

Reply:
<img width="312" alt="image" src="https://github.com/web-platform-tests/wpt/assets/2840106/19bc4156-5370-4a6b-8530-12a6d37225a2">

Reply:
And thanks for the explanation!

Reply:
Oh I see but it's indeed broken: `FAIL message: webdriver.bidi.modules.script.ScriptEvaluateResultException: SyntaxError: Unexpected token '{'`. But yes, let me fix that separately given that this is already broken and cannot be found by the linter.

Reply:
Feel free to push it on top of this PR. I am not planning to make any other changes to it.

Reply:
I'll fix it via https://bugzilla.mozilla.org/show_bug.cgi?id=1864871.